x/consensus/keeper/consensus/consensus_test.go

```diff
@@ -42,7 +44,7 @@ func TestConsensusQueueAllMethods(t *testing.T) {
 			QueueTypeName:         "simple-message",
 			Sg:                    sg,
 			Ider:                  keeperutil.NewIDGenerator(sg, nil),
-			Cdc:                   types.ModuleCdc,
+			Cdc:                   nil,
```

Comment:
Why is this `nil` now? I'm pretty sure that will break production and test code.

Reply:
Resolved